magefiles/magefile.go

```diff
@@ -567,8 +655,93 @@ func pwd() string {
 	return wd
 }
 
+// Install the specified version of porter
+func NewEnsurePorterAt(version string) {
```

Comment:
Can you remove this implementation and use the one in the magefiles repository? https://github.com/getporter/magefiles/blob/0b4ba70c8f969d7dfdc489310ab94a0dbb06a390/porter/porter.go#L27